api/v1beta1/openstackbaremetalset_types.go

```diff
@@ -29,7 +39,15 @@ type OpenStackBaremetalSetSpec struct {
 	BaremetalHosts map[string]string `json:"baremetalHosts"`
 	// RhelImageURL - Remote URL pointing to desired RHEL qcow2 image
 	RhelImageURL string `json:"rhelImageUrl,omitempty"`
+
+	// When set to disabled, automated cleaning will be avoided
+	// during provisioning and deprovisioning.
+	// +optional
+	// +kubebuilder:default:=metadata
```

Comment:
Might be an extra `:` after `default`

Reply:
The doc asks for it seems. Anyway I did not see it anywhere used, so changed. https://book.kubebuilder.io/reference/markers/crd-validation.html